zaplings/templates/zaplings/creatorsnight.html

```diff
@@ -162,7 +162,8 @@ <h5 class="inline-title add-margin-top"></h5><h4 class="inline-title add-margin-
                 </div>
             </div>
 
-            <form id="rsvp-basic" name="rsvp-basic">
+           <form id="rsvp-basic" name="rsvp-basic" action="{% url 'zaplings:process-rsvp' %}" method="post"/>
```

Comment:
@nickolashe I found my error. My terminal was on the master branch when I thought it was on danny. So I kept pulling from danny to master instead of vice versa before committing my changes. Apologies for the extra work it created for you.